### DISCUSSION TITLE
I think it is a wrong behavour

### DISCUSSION THREAD

**Discussion by butschster** [spatie/ray#544] [Show and tell]
Hi everyone!
I'm trying to create a web application for `spatie/ray` package (Ray server) and I bumped into a problem.

I think there is a mistake in a code from given link https://github.com/spatie/ray/blob/main/src/Client.php#L51

Maybe there should be something like this
```php
$success = curl_errno($curlHandle) !== CURLE_HTTP_NOT_FOUND;
```

**Comment by sinnbeck** [comment]
It seems correct actually. When you call the url with ray closed, you will get a "Site cant be reached"
http://localhost:23517/_availability_check

If you then open Ray and call the same url, you will actually get a 404 (not found) response. So checking if the error is "Not found" response, seems to be correct behavior

**Reply by butschster** [reply]
Got it. Thank you.
